### ISSUE TITLE
Raw complex I/Q export

### ISSUE THREAD

**Issue by azonenberg** [azonenberg/scopehal#549]

**Comment by dkozel** [comment]
As a thought, the SigMF metadata format might be useful to you here. It's specifically designed for allowing a raw datafile to be stored with a separate simple metadatafile alongside describing the properties of the recording. So no interference with inport/export with software that already/only supports raw files, no header or etc to mess with parsing and updating while loading/writing captures, you can use optimized/specific code to handle the data files and leave the metadata to the side. It also specifically handles multiple simultaneous streams, segmented captures, and signal annotations.

https://sigmf.org

If you do think this is interesting and have any questions I can possibly answer or link you with the spec authors.

**Comment by 777arc** [comment]
yeah would be sweet if SigMF was supported, and should be very little extra work needed since the data file is just a binary file anyway
